sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/CallingServerBaseEvent.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+/** The base event interface. */
+public interface CallingServerBaseEvent {
```

Comment:
CallAutomationEventBase, to align with .NET and CallAutomation naming

Reply:
fixed